src/igvfd/schemas/sequence_data.json

```diff
@@ -0,0 +1,71 @@
+{
+    "title": "Sequence data",
+    "$id": "/profiles/sequence_data.json",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "required": [
+        "award",
+        "lab"
+    ],
+    "identifyingProperties": [
+        "uuid",
+        "aliases",
+        "md5sum"
+    ],
+    "additionalProperties": false,
+    "mixinProperties": [
+        {
+            "$ref": "file.json#/properties"
+        },
+        {
+            "$ref": "mixins.json#/basic_item"
+        },
+        {
+            "$ref": "mixins.json#/accession"
+        },
+        {
+            "$ref": "mixins.json#/attribution"
+        },
+        {
+            "$ref": "mixins.json#/documents"
+        }
+    ],
+    "dependentSchemas": {
+        "$merge": [
+            "file.json#/dependentSchemas"
+        ]
+    },
+    "type": "object",
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "content_type": {
```

Comment:
I know that it may be very long, but perhaps we should add a comment with an explanation of what each of the enums is? with examples?